frontends/.yarnrc.yml

```diff
@@ -0,0 +1,19 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.1.1.cjs
```

Comment:
After `docker compose up`, I see `frontends/.yarn/install-state.gz` as an untracked file.

Reply:
Added to the ignore file.